content/attacks/posts/2018-09-14-Zaif.md

```diff
@@ -0,0 +1,30 @@
+---
+date: 2018-09-14
+categories: "Custodian"
+title: "Zaif loses $60 Million in Cyberattack"
```

Comment:
Please, reformat headers according to [new changes](Please, reformat headers according to [new changes]) and merge [main](https://github.com/1712n/dn-institute/tree/main) into the branch